packages/reactive-element/src/decorators/query-assigned-elements.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * IMPORTANT: For compatibility with tsickle and the Closure JS compiler, all
+ * property decorators (but not class decorators) in this file that have
+ * an @ExportDecoratedItems annotation must be defined as a regular function,
+ * not an arrow function.
+ */
+
+import {ReactiveElement} from '../reactive-element.js';
+import {decorateProperty} from './base.js';
+
+export interface QueryAssignedElementsOptions extends AssignedNodesOptions {
+  /**
+   * A string name of the slot. Leave empty for the default slot.
+   */
+  slot?: string;
+  /**
+   * A string which filters the results to elements that match the given css
```

Comment:
capitalize "CSS"

Reply:
I don't think you need the phrase with the type like "A string...", I think you can say "A CSS selector used to filter the elements returned..."

Reply:
Done. Great comment.